### DISCUSSION TITLE
Documentation improvements on unpoly.com

### DISCUSSION THREAD

**Discussion by triskweline** [unpoly/unpoly#385] [Announcements]
I made some changes to Unpoly's [documentation site](https://unpoly.com):

- You can run a full text search on all site content (not just the title)
- The navigation tree also lists topic pages that were formerly only linked within the page content, like [Opening Overlays](https://unpoly.com/opening-overlays) or [Navigation](https://unpoly.com/navigation).
- The navigation lists are now grouped by type: Topics / HTML selectors / Events / JavaScript / HTTP.
- You can now focus the search box using the `/` (slash) key.

I hope this makes the documentation content more accessible.

![New navigation tree](https://user-images.githubusercontent.com/89953/169544377-ac60a0da-79e4-41e4-8804-68ecf7b9b82e.gif)

**Comment by adam12** [comment]
This is great! And actually addresses one of the pain points I had with the documentation, where I could only find some pages through a magic traversal of the website. Having them in the sidebar looks like it will be a big help!

**Reply by roscrl** [reply]
Love how simple the [implementation](https://github.com/unpoly/unpoly-site/blob/master/source/javascripts/components/content_search.js.erb) is too!
